packages/@glimmer/runtime/lib/vm/element-builder.ts

```diff
@@ -214,15 +216,31 @@ export class NewElementBuilder implements ElementBuilder {
     element: SimpleElement,
     guid: string,
     insertBefore: Maybe<SimpleNode>
-  ): Nullable<RemoteLiveBlock> {
-    return this.__pushRemoteElement(element, guid, insertBefore);
+  ): RemoteLiveBlock {
+    const block = this.__pushRemoteElement(element, guid, insertBefore);
+    if (this.env.debugRenderTree) {
+      const namedArgs: Record<string, Reference> = {};
+      if (insertBefore) {
```

Comment:
I'll see if I can figure out how we can use the proper utilities so we can get rid of the type cast below; but by the way, we shouldn't guard it like this – it maybe fine to omit `undefined` (or we can just let inspector make the choice), but for `in-element`, `null` and `undefined are specifically different – `undefined` clears the element and `null` inserts it at the end of the children (ie append), so we would want to preserve that